src/set_theory/surreal/basic.lean

```diff
@@ -284,9 +294,28 @@ noncomputable instance : linear_ordered_add_comm_group surreal :=
   decidable_le := classical.dec_rel _,
   ..surreal.ordered_add_comm_group }
 
+end surreal
+
+open surreal
+
+namespace ordinal
+
+/-- Converts an ordinal into the corresponding surreal. -/
+noncomputable def to_surreal (o : ordinal) : surreal :=
+mk _ (numeric_to_pgame o)
+
+@[simp] theorem to_surreal_le_iff (a b : ordinal) : a.to_surreal ≤ b.to_surreal ↔ a ≤ b :=
+to_pgame_le_iff
+
+@[simp] theorem to_surreal_lt_iff (a b : ordinal) : a.to_surreal < b.to_surreal ↔ a < b :=
+to_pgame_lt_iff
+
+@[simp] theorem to_surreal_inj (a b : ordinal) : a.to_surreal = b.to_surreal ↔ a = b :=
+by { change ⟦_⟧ = ⟦_⟧ ↔ _, rw quotient.eq, exact to_pgame_equiv_iff }
```

Comment:
```suggestion
@[simp] theorem to_surreal_inj {a b : ordinal} : a.to_surreal = b.to_surreal ↔ a = b :=
by { change ⟦_⟧ = ⟦_⟧ ↔ _, rw quotient.eq, exact to_pgame_equiv_iff }
```
Also would be good to have the `injective` spelling too

Reply:
I've instead made `to_surreal` an order embedding, which circumvents the need for these lemmas.